CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing to UNCode
+We love your input! We want to make contributing to this project as easy and 
+transparent as possible, whether it's:
+
+    - Reporting a bug.
+    - Discussing the current state of the code.
+    - Submitting a fix.
+    - Proposing new features
+
+## Setup development environment
+
+Follow this [tutorial/steps][development_url] on how to setup the development environment and be able to contribute.
+
+## Our Development Process
+
+We use github to host code, to track issues and feature requests, as well 
+as accept pull requests.
+
+## Your First Contribution
+
+Unsure where to begin contributing to UNCode? You can start by looking through these beginner issues:
+**Good first issue** - issues which should only require a few lines of code and a not big understanding of UNCode.
+
+Working on your first Pull Request? You can learn how from this *free* series, 
+[How to Contribute to an Open Source Project on GitHub][contribute_github_url].
+
+## Pull Requests
+
+Pull requests are the best way to propose changes to the codebase 
+(we use [Github Flow][github_flow_url]. 
+
+We actively welcome your pull requests.
+
+1.Fork the repo (in case not an admin) and create your branch from `master`.
+2.If you've added code that should be tested, add tests.
+3.If you've changed APIs, update the documentation.
+4.Ensure the test suite passes.
+5.Be sure you have followed the code style for the project.
+6.If you haven't already, complete the Contributor License Agreement ("CLA"). Note the code of conduct.
+7.Create the pull request to `master` branch.
+
+### Code review
+
+UNCode team will check your pull request and one of us will be assigned to review it and
+leave feedback. Make to reply the given feedback ASAP.
+
+## Contributor License Agreement ("CLA")
+In order to accept your pull request, you will need to sign the Contributor License Agreement.
+Once the PR is created, you will be told whether or not you need to sign it and how to do it.
+
+## Issues
+We use GitHub issues to track public bugs or Feature Requests. Please ensure your description is
+clear and has sufficient instructions to be able to reproduce the issue.
+
+## License
+In short, when you submit code changes, your submissions are understood to be under the 
+same [GNU AGPL v3][licence_url] that covers the project. 
```

Comment:
Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/JuezUN/INGInious/pullRequest?prid=6256833)